app/src/main/java/org/wikipedia/descriptions/DescriptionEditReadArticleBarView.kt

```diff
@@ -36,7 +41,29 @@ class DescriptionEditReadArticleBarView @JvmOverloads constructor(
             viewArticleImage!!.visibility = VISIBLE
             viewArticleImage!!.loadImage(Uri.parse(pageSummary.thumbnailUrl))
         }
+        viewImageThumbnail.visibility = View.GONE
+        viewReadButton.visibility = View.VISIBLE
+        show()
+    }
+
+    fun setImageDetails(thumbUrl: String, @NonNull fileName: String) {
+        setConditionalLayoutDirection(this, WikipediaApp.getInstance().language().appLanguageCode)
+        viewArticleTitle!!.text = StringUtil.fromHtml(fileName)
+
+        val padding: Int = DimenUtil.dpToPx(8.0f).toInt()
+        viewArticleImage!!.visibility = VISIBLE
+        viewArticleImage!!.setPadding(padding, padding, padding, padding)
+        viewArticleImage!!.setColorFilter(ResourceUtil.getThemedColor(context, R.attr.main_toolbar_icon_color))
+        viewArticleImage.setImageResource(R.drawable.ic_keyboard_arrow_up_black_24dp)
```

Comment:
Please use `setActualImageResource` instead of `setImageResource`

Reply:
Thank you!! I forgot to look for an alternate for the deprecated api..

Reply:
it is not rendering the resource for both setActualImageResource and loadImage(int id). May be something to discuss

Reply:
I looked at the Zeplin design and compared with the actual implementation, the "arrow" image does not have the same position as the image thumbnail position. It should be an individual `ImageView`. 

https://app.zeplin.io/project/57a120b91998d8977642a238/screen/5cdac7fc25662e679ce616a2

Reply:
for the article it is thumnail-title-'READ' button, and for image caption it is arrow-title-thumbnail where the button is.. so Why make an additional imageView?

Reply:
Not actually related to this thread, but I am seeing the "arrow" icon has too many "left" margin than the design. The arrow should remain in the same position as the arrow in `Image Preview Dialog`